### PR TITLE
python3.5+

### PR DESCRIPTION
**Pull request by bqpd**
removes `mosek` solver as well, though not `mosek_cli` just yet

### CODE REVIEW

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
can't recreate the pylint failures on macys_vm because I have different (lower) versions of python+pylint on my computer.

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
for some reason Jenkins doesn't seem to be pulling down the latest test scripts from acdl.mit.edu, which should fix macys_vm's units tests by enforcing a lower pint version. will take a look tomorra

**Comment by bqpd** [comment]
(newer versions of Python3 have determinate dictionary ordering as well, in a which which makes many of my sorts added to make python3 pass tests unnecessary...)

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

File: gpkit/tests/t_keydict.py [hunk 1]
```diff
@@ -39,12 +39,6 @@ def test_failed_getattr(self):
         self.assertEqual(res, 5)
         self.assertIn("waldo", kd)
 
-    def test_dictlike(self):
-        kd = KeyDict()
-        kd["a string key"] = "a string value"
-        self.assertTrue(isinstance(kd, dict))
-        self.assertEqual(list(kd.keys()), ["a string key"])
-
```
Comment (bqpd) [inline gpkit/tests/t_keydict.py]: we don't really have a need to target using KeyDicts for things without a `keys` attribute

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test this please

**Comment by bqpd** [comment]
test models please

**Comment by galbramc** [comment]
test this please

**Comment by galbramc** [comment]
test this please

**Comment by galbramc** [comment]
test this please

**Comment by galbramc** [comment]
test this please

**Comment by bqpd** [comment]
hey look at that! unit tests passing!

**Comment by galbramc** [comment]
This is bad
```
+++ python -c '__import__("mosek").Env()'
/jenkins/workspace/CE_gpkit_PR_pylint/buildnode/reynolds/JenkinsGPkit/mosek_installs.sh: line 16: 28144 Bus error               (core dumped) python -c '__import__("mosek").Env()'
```

**Comment by galbramc** [comment]
I had to switch from "Publish xUnit" to "Publish JUnit". I need you to make that change on all the jenkins jobs.

**Comment by bqpd** [comment]
Where did you make that switch? Are those different jenkins plugins?

**Comment by galbramc** [comment]
It's in the "Post-build Actions" of the configuration. You just have to swap the settings. It should look like this.

Screen Shot 2020-03-02 at 10 11 30 PM

**Comment by galbramc** [comment]
The old settings you need to remove looks like this.

Screen Shot 2020-03-02 at 10 12 34 PM

**Comment by bqpd** [comment]
test models please

**Comment by bqpd** [comment]
@galbramc just updated the other Jenkins tests, thanks!

**Comment by bqpd** [comment]
test models please

**Comment by bqpd** [comment]
test models please

**Comment by bqpd** [comment]
test models please

**Pull request merged**